pages/shared/data/tools.yaml

```diff
@@ -190,6 +190,16 @@ platforms:
       height: 691
     formats:
       - email
+  - name: Outlook.com AMP Docs
```

Comment:
Would it be OK to use "Outlook.com AMP documentation"? I think we try to about abbreviations like docs when possible.